### PR TITLE
python3Packages.fnvhash: init at 0.1.0

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python FNV hash implementation.

https://github.com/znerol/py-fnvhash

This is a Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/home-assistant/component-packages.nix [hunk 1]
```diff
@@ -350,7 +350,7 @@
     "hlk_sw16" = ps: with ps; [ ]; # missing inputs: hlk-sw16
     "home_connect" = ps: with ps; [ aiohttp-cors ]; # missing inputs: homeconnect
     "homeassistant" = ps: with ps; [ ];
-    "homekit" = ps: with ps; [ HAP-python pyqrcode aiohttp-cors ha-ffmpeg zeroconf ]; # missing inputs: PyTurboJPEG base36 fnvhash
+    "homekit" = ps: with ps; [ HAP-python pyqrcode aiohttp-cors fnvhash ha-ffmpeg zeroconf ]; # missing inputs: PyTurboJPEG base36
```
Comment (SuperSandro2000) [inline pkgs/servers/home-assistant/component-packages.nix:353]: base36 is already in master.

**Comment by SuperSandro2000** [comment]
@fabaff please fix the merge conflict

File: pkgs/servers/home-assistant/component-packages.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/servers/home-assistant/component-packages.nix]: Yes, it was introduced with https://github.com/NixOS/nixpkgs/pull/109530

**Comment by fabaff** [comment]
@SuperSandro2000, thanks. Conflict is resolved now.

**Comment by mweinelt** [comment]
Uh, now you seem to have squashed everything into one commit.

https://github.com/NixOS/nixpkgs/pull/109535/commits/596f4bb6eda0e45856b65be267e5c6e2df0b2fef

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109535` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python37Packages.fnvhash</li>
<li>python38Packages.fnvhash</li>
<li>python39Packages.fnvhash</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 109535` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python37Packages.fnvhash</li>
    <li>python38Packages.fnvhash</li>
    <li>python39Packages.fnvhash</li>
  </ul>
</details>

**Comment by fabaff** [comment]
> Uh, now you seem to have squashed everything into one commit.

Thanks @mweinelt

**Review by mweinelt — APPROVED** [review]

**Pull request merged**